infra/chromadb_compaction/lambdas/enhanced_compaction_handler.py

```
@@ -0,0 +1,1557 @@
+    # Process stream messages if any
+    if stream_messages:
+        result = process_stream_messages(stream_messages)
+        if OBSERVABILITY_AVAILABLE:
+            logger.info("Processed stream messages", count=len(stream_messages))
+        else:
+            logger.info("Processed %d stream messages", len(stream_messages))
+
+    # Process delta messages if any - collect failed message IDs
+    if delta_message_records:
+        delta_bodies = [msg["body"] for msg in delta_message_records]
+        delta_result = process_delta_messages(delta_bodies)
```

Ruff (0.12.2)

Local variable `delta_result` is assigned to but never used

Remove assignment to unused variable `delta_result`

(F841)